lms/envs/test.py

```diff
@@ -26,6 +26,11 @@
 
 from openedx.core.lib.tempdir import mkdtemp_clean
 
+# This patch disabes the commit_on_success decorator during tests
+# in TestCase subclasses.
+from util.testing import patch_testcase
```

Comment:
This import seems ambiguous. Is there any chance it could be getting some other `util` module from somewhere else? Can we qualify this import somehow?